internal/seclang/parser.go

```diff
@@ -165,16 +164,16 @@ func (p *Parser) evaluateLine(l string) error {
 	}
 
 	p.options.Opts = opts
-	p.options.Config.Set("last_profile_line", p.currentLine)
```

Comment:
Yikes, it seems this was set as `last_profile_line` and gotten with `parser_last_line` IIUC